app/Resources/views/page.html.twig

```diff
@@ -163,6 +163,8 @@
         };
 
         {% if gtm_id %}
+        var gtmDataLayer = window.gtmDataLayer = window.gtmDataLayer || [];
```

Comment:
Unclear whether terse or an error. If intentional, should be split into more than one expression for clarity.